src/app/Http/Controllers/Auth/UserController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\UserRequest;
+use App\Models\User;
+use Illuminate\Contracts\View\View;
+
+/**
+ * UserController
+ */
+class UserController extends Controller
+{
+    /**
+     * Constructor
+     *
+     * @return void
+     */
+    public function __construct (
+        public User $user,
+    ) {}
+    
+    /**
+     * ユーザ登録画面表示
+     *
+     * @return View
+     */ 
+    public function register(): View
+    {
+        return view('auth.user-register');
+    }
+
+    /**
+     * ユーザ登録処理
+     *
+     * @param UserRequest $request
+     * @return View
+     */
+    public function completeRegister(UserRequest $request): View
+    {
+        $values = $request->validated();
+        
+        $this->user->create($values);
```

Comment:
`$values`定義しなくても良いかな！
変数定義するケースは下記の場合くらいな気がする。
- 二箇所以上で定義する時
- 変数のデータが別のインスタンスに変わる時
   - ex) `$post = $user->post` のようなuserインスタンスからpostのインスタンスへ変わる場合は、データ型が変わったことを読み手に伝えることで、可読性向上を図るため
- 複雑なソースコードをシンプルにしたい時

```suggestion
        $this->user->create($request->validated());
```

Reply:
修正済み